server/routes/session.js

```diff
@@ -157,7 +157,7 @@ const featuredSessions = [
 
 // GET all sessions
 router.get('/', function (req, res, next) {
-    if (req.query) {
+    if (req.query.sport) {
         let filteredSessions = sessions.filter(session => session.sport === req.query.sport)
         return res.status(200).send(filteredSessions)
     } 
```

Comment:
I get that you've just updated a null check and didn't actually just add this stuff in, but I think I missed when this was originally added. Just wondering, why are we filtering by sport in the get all sessions endpoint? Shouldn't it return all sessions like the comment suggests?

Reply:
😭 🙇🏼